index.js

```diff
@@ -325,6 +326,14 @@ Sharp.prototype.gamma = function(gamma) {
 };
 
 /*
+  Normalize histogram
+*/
+Sharp.prototype.normalize = function(normalize) {
+  this.options.normalize = (typeof normalize === 'boolean') ? normalize : true;
+  return this;
+};
+
```

Comment:
Please can you add `Sharp.prototype.normalise = Sharp.prototype.normalize;` to keep with the greyscale/grayscale pattern of supporting both types of English :)